### DISCUSSION TITLE
HA can't find any HM devices expect the CCU3

### DISCUSSION THREAD

**Discussion by slackstarr** [SukramJ/aiohomematic#790] [Q&A]
Hey folks,

my brain is quite empty and I am very frustrated. I successfully installed HACS, the additional Homematic(IP) Local repo and I was able to configure the CCU3.

Unfortunately HA can't "auto discover" any Homematic devices. I am aware that HA (running on a Raspberry Pi4 B) needs to able to "communicate" with the CCU3. I am still in a learning phase but came to a point where I don't know what to try next.

It would be awesome if you could help me.

Some details regarding my setup:

Raspberry Pi 4 Model B, 8GB Ram
CCU3 firmware: 3.65.11
Home Assistant 2022.12.8
Homematic(IP) Local: 1.25.1

HA Log:

Logger: hahomematic.json_rpc_client
Source: runner.py:120
First occurred: 18:56:43 (96 occurrences)
Last logged: 18:58:32

_do_login failed: Unable to open session: {'name': 'JSONRPCError', 'code': 501, 'message': 'invalid credentials or too many sessions'}
_post_script failed: Error while logging in via JSON-RPC.
_post failed: Error while logging in via JSON-RPC.

Logger: hahomematic.central_unit
Source: runner.py:120
First occurred: 18:56:43 (32 occurrences)
Last logged: 18:58:32

check_connection failed: No clients exist. Trying to create clients for server ccu3-webui
_create_clients failed: Interface: VirtualDevices is not available for backend.
_create_clients failed: Interface: HmIP-RF is not available for backend.
_create_clients failed: Interface: BidCos-RF is not available for backend.

It would be great if someone could help!

Many thanks!
/Marco

**Comment by slackstarr** [comment]
I think I solved the issue by reading what I've written in my post. It's always the user's fault, right? ;-)

**Reply by SukramJ** [reply]
This is not an answer.
You should write down for others what was the solution for you!

**Reply by slackstarr** [reply]
Hey,

I am aware that my 2nd was not an answer. I was in a hurry and wanted to let others know, that I found a solution. 

My solution can be found in this thread: https://github.com/danielperna84/hahomematic/discussions/792#discussioncomment-4548680
